default.nix

```diff
@@ -545,4 +493,19 @@ in let this = rec {
   project = args: import ./project this (args ({ pkgs = nixpkgs; } // this));
   tryReflexShell = pinBuildInputs ("shell-" + system) tryReflexPackages;
   ghcjsExternsJs = ./ghcjs.externs.js;
+
+  # Deprecated reexports. These were made for `./scripts/*`, but are reexported
+  # here for backwards compatability.
```

Comment:
It might be good to explicitly say that these `inherit`'s will be removed in the future, and perhaps set a timeline (like 6 months) when it might happen by.

Reply:
A related thing is standardizing how we deprecate things. For instance, should it show a warning when you use this attribute in Nix? We also talked about making a flag to remove these deprecations.